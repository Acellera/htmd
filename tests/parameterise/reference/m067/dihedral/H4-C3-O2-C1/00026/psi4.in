set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 -1.276010	 1.052711	 -0.768302
C	 -1.204840	 0.067374	 -0.106123
C	 -2.388195	 -0.672128	 0.458071
O	 -0.067104	 -0.527208	 0.221609
H	 -2.388023	 -1.694060	 0.097175
H	 -3.300579	 -0.174919	 0.164318
H	 -2.315377	 -0.706351	 1.539059
C	 1.170111	 0.034140	 -0.238951
C	 0.902506	 0.533595	 -1.647681
C	 2.265964	 -1.018514	 -0.178355
H	 1.421149	 0.860619	 0.409446
H	 0.105790	 1.268776	 -1.648395
H	 1.795323	 0.995214	 -2.056480
H	 0.612148	 -0.285880	 -2.296377
H	 3.199866	 -0.606571	 -0.547334
H	 2.422722	 -1.351391	 0.839779
H	 2.003796	 -1.874500	 -0.792002

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		8 4 2 1
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "17  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
