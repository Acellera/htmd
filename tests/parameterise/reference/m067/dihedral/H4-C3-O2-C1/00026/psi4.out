    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:35:50 2016

    Process ID:  11254
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 -1.276010	 1.052711	 -0.768302
C	 -1.204840	 0.067374	 -0.106123
C	 -2.388195	 -0.672128	 0.458071
O	 -0.067104	 -0.527208	 0.221609
H	 -2.388023	 -1.694060	 0.097175
H	 -3.300579	 -0.174919	 0.164318
H	 -2.315377	 -0.706351	 1.539059
C	 1.170111	 0.034140	 -0.238951
C	 0.902506	 0.533595	 -1.647681
C	 2.265964	 -1.018514	 -0.178355
H	 1.421149	 0.860619	 0.409446
H	 0.105790	 1.268776	 -1.648395
H	 1.795323	 0.995214	 -2.056480
H	 0.612148	 -0.285880	 -2.296377
H	 3.199866	 -0.606571	 -0.547334
H	 2.422722	 -1.351391	 0.839779
H	 2.003796	 -1.874500	 -0.792002

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		8 4 2 1
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "17  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-3-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:35:50 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O         -1.188305846052     1.129698942254    -0.438863425895    15.994914619560
           C         -1.117135846052     0.144361942254     0.223315574105    12.000000000000
           C         -2.300490846052    -0.595140057746     0.787509574105    12.000000000000
           O          0.020600153948    -0.450220057746     0.551047574105    15.994914619560
           H         -2.300318846052    -1.617072057746     0.426613574105     1.007825032070
           H         -3.212874846052    -0.097931057746     0.493756574105     1.007825032070
           H         -2.227672846052    -0.629363057746     1.868497574105     1.007825032070
           C          1.257815153948     0.111127942254     0.090487574105    12.000000000000
           C          0.990210153948     0.610582942254    -1.318242425895    12.000000000000
           C          2.353668153948    -0.941526057746     0.151083574105    12.000000000000
           H          1.508853153948     0.937606942254     0.738884574105     1.007825032070
           H          0.193494153948     1.345763942254    -1.318956425895     1.007825032070
           H          1.883027153948     1.072201942254    -1.727041425895     1.007825032070
           H          0.699852153948    -0.208892057746    -1.966938425895     1.007825032070
           H          3.287570153948    -0.529583057746    -0.217895425895     1.007825032070
           H          2.510426153948    -1.274403057746     1.169217574105     1.007825032070
           H          2.091500153948    -1.797512057746    -0.462563425895     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.16937  B =      0.06298  C =      0.04928 [cm^-1]
  Rotational constants: A =   5077.59655  B =   1888.13213  C =   1477.30229 [MHz]
  Nuclear repulsion =  328.876427605772335

  Charge       = 0
  Multiplicity = 1
  Electrons    = 56
  Nalpha       = 28
  Nbeta        = 28

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 62
    Number of basis function: 125
    Number of Cartesian functions: 125
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        125     125       0       0       0       0
   -------------------------------------------------------
    Total     125     125      28      28      28       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 258
    Number of basis function: 817
    Number of Cartesian functions: 817
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.3122977662E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -244.12721750467227   -2.44127e+02   8.54169e-02 
   @DF-RHF iter   2:  -231.30621311598378    1.28210e+01   5.89002e-02 DIIS
   @DF-RHF iter   3:  -299.35538019314521   -6.80492e+01   5.79760e-02 DIIS
   @DF-RHF iter   4:  -327.20864442852172   -2.78533e+01   3.45732e-02 DIIS
   @DF-RHF iter   5:  -341.42231418446778   -1.42137e+01   1.72993e-02 DIIS
   @DF-RHF iter   6:  -344.71818533496997   -3.29587e+00   4.51903e-03 DIIS
   @DF-RHF iter   7:  -344.85581095696983   -1.37626e-01   1.77908e-03 DIIS
   @DF-RHF iter   8:  -344.87391232925478   -1.81014e-02   7.37035e-04 DIIS
   @DF-RHF iter   9:  -344.87832270437008   -4.41038e-03   2.16216e-04 DIIS
   @DF-RHF iter  10:  -344.87874701635661   -4.24312e-04   4.66339e-05 DIIS
   @DF-RHF iter  11:  -344.87878360733993   -3.65910e-05   1.53496e-05 DIIS
   @DF-RHF iter  12:  -344.87878794873853   -4.34140e-06   6.13246e-06 DIIS
   @DF-RHF iter  13:  -344.87878853613762   -5.87399e-07   2.81280e-06 DIIS
   @DF-RHF iter  14:  -344.87878866050625   -1.24369e-07   7.25014e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.600713     2A    -20.542587     3A    -11.380576  
       4A    -11.289601     5A    -11.234518     6A    -11.220112  
       7A    -11.184471     8A     -1.472036     9A     -1.368379  
      10A     -1.076685    11A     -1.005055    12A     -0.939133  
      13A     -0.852931    14A     -0.759830    15A     -0.706968  
      16A     -0.685586    17A     -0.652361    18A     -0.611925  
      19A     -0.586474    20A     -0.579796    21A     -0.556837  
      22A     -0.545254    23A     -0.537231    24A     -0.521425  
      25A     -0.499948    26A     -0.482797    27A     -0.444878  
      28A     -0.403433  

    Virtual:                                                              

      29A      0.192207    30A      0.242348    31A      0.259726  
      32A      0.282360    33A      0.291255    34A      0.304387  
      35A      0.315764    36A      0.327540    37A      0.333949  
      38A      0.355380    39A      0.371690    40A      0.383881  
      41A      0.415244    42A      0.422762    43A      0.449130  
      44A      0.500804    45A      0.566604    46A      0.704051  
      47A      0.716095    48A      0.740804    49A      0.757927  
      50A      0.763279    51A      0.786162    52A      0.806666  
      53A      0.838780    54A      0.882769    55A      0.914473  
      56A      0.950370    57A      0.960748    58A      0.974734  
      59A      0.985076    60A      1.037727    61A      1.093642  
      62A      1.115087    63A      1.134056    64A      1.153063  
      65A      1.167062    66A      1.186126    67A      1.192588  
      68A      1.203654    69A      1.211656    70A      1.225347  
      71A      1.236532    72A      1.284049    73A      1.303281  
      74A      1.317405    75A      1.401701    76A      1.416264  
      77A      1.474487    78A      1.556234    79A      1.575551  
      80A      1.648168    81A      1.690492    82A      1.739340  
      83A      1.759292    84A      1.798556    85A      1.871438  
      86A      1.957055    87A      1.977805    88A      2.034648  
      89A      2.049624    90A      2.097878    91A      2.150873  
      92A      2.171847    93A      2.202248    94A      2.225098  
      95A      2.267742    96A      2.303232    97A      2.326832  
      98A      2.372854    99A      2.398470   100A      2.458701  
     101A      2.498979   102A      2.539081   103A      2.574244  
     104A      2.598634   105A      2.610715   106A      2.651606  
     107A      2.738121   108A      2.776714   109A      2.813025  
     110A      2.833438   111A      2.900976   112A      2.998913  
     113A      3.063977   114A      3.077234   115A      3.187527  
     116A      3.375448   117A      3.425264   118A      3.572616  
     119A      4.455666   120A      4.535247   121A      4.618599  
     122A      4.670111   123A      4.755490   124A      4.865193  
     125A      4.992865  

    Final Occupation by Irrep:
              A 
    DOCC [    28 ]

  Energy converged.

  @DF-RHF Final Energy:  -344.87878866050625

   => Energetics <=

    Nuclear Repulsion Energy =            328.8764276057723350
    One-Electron Energy =               -1122.0274762055385054
    Two-Electron Energy =                 448.2722599392600387
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -344.8787886605061885

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     4.1510      Y:    -2.6209      Z:    -0.9336

  Electronic Dipole Moment: (a.u.)
     X:    -3.9633      Y:     2.0524      Z:     1.3680

  Dipole Moment: (a.u.)
     X:     0.1877      Y:    -0.5685      Z:     0.4344     Total:     0.7397

  Dipole Moment: (Debye)
     X:     0.4771      Y:    -1.4450      Z:     1.1041     Total:     1.8801


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-3-4.cx1.hpc.ic.ac.uk at Tue Jul  5 16:36:01 2016
Module time:
	user time   =      10.81 seconds =       0.18 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         11 seconds =       0.18 minutes
Total time:
	user time   =      10.81 seconds =       0.18 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         11 seconds =       0.18 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     4.1510      Y:    -2.6209      Z:    -0.9336

  Electronic Dipole Moment: (a.u.)
     X:    -3.9633      Y:     2.0524      Z:     1.3680

  Dipole Moment: (a.u.)
     X:     0.1877      Y:    -0.5685      Z:     0.4344     Total:     0.7397

  Dipole Moment: (Debye)
     X:     0.4771      Y:    -1.4450      Z:     1.1041     Total:     1.8801

  Quadrupole Moment: (Debye Ang)
    XX:   -39.4364     YY:   -45.8723     ZZ:   -43.7387
    XY:     3.5044     XZ:    -1.7908     YZ:     1.9907

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     3.5794     YY:    -2.8565     ZZ:    -0.7229
    XY:     3.5044     XZ:    -1.7908     YZ:     1.9907

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     O     4.29304  4.29304  0.00000 -0.58607
       2     C     2.60617  2.60617  0.00000  0.78766
       3     C     3.28065  3.28065  0.00000 -0.56130
       4     O     4.31834  4.31834  0.00000 -0.63669
       5     H     0.40017  0.40017  0.00000  0.19966
       6     H     0.39719  0.39719  0.00000  0.20562
       7     H     0.39979  0.39979  0.00000  0.20041
       8     C     2.91734  2.91734  0.00000  0.16532
       9     C     3.28398  3.28398  0.00000 -0.56796
      10     C     3.24275  3.24275  0.00000 -0.48551
      11     H     0.41341  0.41341  0.00000  0.17317
      12     H     0.35388  0.35388  0.00000  0.29224
      13     H     0.42737  0.42737  0.00000  0.14527
      14     H     0.42462  0.42462  0.00000  0.15076
      15     H     0.41444  0.41444  0.00000  0.17112
      16     H     0.41306  0.41306  0.00000  0.17388
      17     H     0.41379  0.41379  0.00000  0.17242

   Total alpha = 28.00000, Total beta = 28.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
